dll/win32/uxtheme/CMakeLists.txt

```diff
@@ -12,6 +12,7 @@ list(APPEND SOURCE
     ncscrollbar.c
     nonclient.c
     property.c
+    resource.h
```

Comment:
Why compile `resource.h`? I don't think precompiled header is useful in this situation.

Reply:
`uxthemep.h` is also included here, isn't this also for dependency tracking?

Reply:
Yes, it's for dependency tracking, see other modules:
- https://git.reactos.org/?p=reactos.git;a=blob;f=base/applications/mstsc/CMakeLists.txt;hb=e0b67a29901cfe1ef11608fd72a24c48aa9525f1
- https://git.reactos.org/?p=reactos.git;a=blob;f=base/applications/rapps/CMakeLists.txt;hb=e0b67a29901cfe1ef11608fd72a24c48aa9525f1